scripts/vmpublish.rb

```diff
@@ -0,0 +1,80 @@
+require 'logger'
+require 'pathname'
+require 'trollop'
+require 'json'
+
+require_relative 'config'
+
+
+def find_imgfac_image(image)
+  f = File.new(image)
+  Dir.chdir(STORAGE_DIR) do
+    Dir.glob('*.body') do |filename|
+      return filename.split('.').first if File.new(filename).stat.ino == f.stat.ino
+    end
+  end
+end
+
+
+def upload_imgfac_image(uuid, target, reference, version)
+  if target == 'vagrant'
+    creds_file = CREDS_DIR.join('atlas.json')
+    params = "atlas @ignored #{creds_file} --id #{uuid}"
```

Comment:
As mentioned above, we'll need instructions on installing atlas in the README, same for gce below.